lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -655,6 +657,30 @@ def test_html_view_for_invalid_certificate(self):
         self.assertIn("Cannot Find Certificate", response.content)
         self.assertIn("We cannot find a certificate with this URL or ID number.", response.content)
 
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_html_lang_attribute_is_dynamic_for_invalid_certificate_html_view(self):
+        """
+        Tests that Certificate HTML Web View's lang attribute is based on user language.
+        """
+        self._add_course_certificates(count=1, signatory_count=2)
+        test_url = get_certificate_url(
+            user_id=self.user.id,
+            course_id=unicode(self.course.id)
+        )
+
+        # invalidate certificate and verify that "Cannot Find Certificate" is returned
+        self.cert.invalidate()
+
+        user_language = 'fr'
+        self.client.cookies[settings.LANGUAGE_COOKIE] = user_language
```

Comment:
I wonder if `translation.override` wasn't working because we hadn't created a `DarkLangConfig` for this test?
 I'm curious how setting the cookie works without that config, as the DarkLang middleware should only allow released languages to be resolved from the cookie. See https://github.com/edx/edx-platform/blob/master/lms/djangoapps/certificates/tests/test_webview_views.py#L1118 for an example DarkLangConfig.

Reply:
I spoke with Cale about DarkLang, and learned that it's not necessary for these tests. Please disregard my comment. Your test does not need to change.